README.md

```diff
@@ -107,6 +107,11 @@ variable "application_cluster_ssl_policy" {
   description = "Policy for SSL"
   default = "ELBSecurityPolicy-TLS-1-2-2017-01"
 }
+
+variable "application_cluster_launch_configuration_detailed_monitoring" {
```

Comment:
I'd very appreciate some description, what the configuration option means. What exactly means "detailed monitoring"?

Reply:
@michz i added a link to the aws documentation